flink-ml-python/pyflink/ml/recommendation/tests/test_swing.py

```diff
@@ -104,6 +105,7 @@ def test_param(self):
         self.assertEqual(5, swing.alpha1)
         self.assertEqual(1, swing.alpha2)
         self.assertAlmostEqual(0.35, swing.beta, delta=1e-9)
+        self.assertEqual(1, swing.seed)
```

Comment:
Let's also check the default value of `seed`.

Reply:
OK. The value of seed and different output while setting non-default value are tested in latest commit.